src/LifxNet/LifxClient.cs

```diff
@@ -1,300 +1,253 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+using System.Diagnostics;
 using System.IO;
-using System.Text;
-using System.Threading.Tasks;
-using System.Net.Sockets;
+using System.Linq;
 using System.Net;
+using System.Net.Sockets;
+using System.Threading.Tasks;
 
-namespace LifxNet
-{
+namespace LifxNet {
 	/// <summary>
 	/// LIFX Client for communicating with bulbs
 	/// </summary>
-	public partial class LifxClient : IDisposable
-	{
-		
+	public partial class LifxClient {
 		private const int Port = 56700;
-		private UdpClient? _socket;
-        private bool _isRunning;
+		private readonly UdpClient _socket;
+		private bool _isRunning;
 
-		private LifxClient()
-		{
+		private LifxClient() {
+			IPEndPoint end = new IPEndPoint(IPAddress.Any, Port);
+			_socket = new UdpClient(end) {Client = {Blocking = false}, DontFragment = true};
```

Comment:
`DontFragment = true` throws a `NotSupported` exception with .NET 5 on MacOs Big Sur. Setting the property to false seems to cure the issue.